Package.swift

```diff
@@ -8,13 +8,14 @@ let package = Package(
         .package(url: "https://github.com/ReactiveCocoa/ReactiveSwift.git", from: "6.0.0"),
         .package(url: "https://github.com/Babylonhealth/ReactiveFeedback", from: "0.7.0"),
         .package(url: "https://github.com/Quick/Nimble.git", from: "8.0.2"),
-        .package(url: "https://github.com/krzyzanowskim/CryptoSwift.git", from: "1.3.0")
+        .package(url: "https://github.com/krzyzanowskim/CryptoSwift.git", from: "1.3.0"),
```

Comment:
Leftover of modifying the Package.swift while trying to add common test target framework…

```suggestion
        .package(url: "https://github.com/krzyzanowskim/CryptoSwift.git", from: "1.3.0")
```